### ISSUE TITLE
[A11Y] There's no alternative to the slider gesture

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ods-ios#734]
Slider gesture should have an alternative with -/+ button

![image](https://github.com/Orange-OpenSource/ods-ios/assets/105045667/565aafcc-dff7-49a3-85ee-b7e63381ca12)
